packages/harpguru-core/src/components/menu-of-tunings/hooks/use-zoom-items/use-zoom-items.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+
+import { OptionItem } from '../../../option-item'
+import type { OptionItemProps } from '../../../option-item'
+import { ZoomIds } from '../../../../types'
+import type { UseGlobal } from '../../../../types'
+
+type ItemCallback = ZoomIds
+type ItemTapHandler = (arg0: ZoomIds) => void
+
+type ValvingItems = ReadonlyArray<
+  React.ReactElement<OptionItemProps<ItemCallback>>
+>
+
+export const useZoomItems = (
+  useGlobal: UseGlobal,
+  itemTapHandler: ItemTapHandler
+): ValvingItems => {
+  const [columnBounds] = useGlobal('columnBounds')
+  const zoomIds = [ZoomIds.Fit, ZoomIds.Seven]
+  const activeZoom = (() => {
+    if (columnBounds === 'FIT') return ZoomIds.Fit
```

Comment:
Probably need a `useDetermineZoomId` hook which might need to be in a shared location, as well as a more local hook to map the various zoom levels to the relevant text in various places including here (the items in the list), as well as the subtitle of the menu.

Reply:
The right solution in the end was to use a util which was useful in multiple locations. All it needs to be passed is the columnBounds.